katas/content/multi_qubit_systems/bell_state_change_1/solution.md

```diff
@@ -0,0 +1,21 @@
+We recognize that the goal is another Bell state. In fact, it is one of the four Bell states.
+
+We remember from "Sign flip" that the Pauli Z gate will change the state of the $|0\rangle$ basis state of a single qubit, so this gate seems like a good candidate for what we want to achieve. This gate leaves the sign of the $|0\rangle$ basis state of a superposition unchanged, but flips the sign of the $|1\rangle$ basis state of the superposition.
```

Comment:
We need to take care with references here, since those tasks are in a different kata now. We can refer to the whole kata
```suggestion
We remember from the Single-Qubit Gates kata that the Pauli Z gate will change the state of the $|1\rangle$ basis state of a single qubit, so this gate seems like a good candidate for what we want to achieve. This gate leaves the sign of the $|0\rangle$ basis state of a superposition unchanged, but flips the sign of the $|1\rangle$ basis state of the superposition.
```